docs/notebooks/dispersion/two_fluid_dispersion_solver.ipynb

```diff
@@ -0,0 +1,271 @@
+{
```

Comment:
<p>The <code>text</code>  variable should be replaced with <code>text = f"$v_A/c_s = {v_A/c_s:.1f}, c/v_A = 10^{np.log10(c/v_A):.0f}, \\theta = {theta.value:.0f}" + "^{\\circ}$"</code> . </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/932/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>